lib/node_modules/@stdlib/math/base/special/cosc/test/fixtures/julia/runner.jl

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env julia
+#
+# @license Apache-2.0
+#
+# Copyright (c) 2018 The Stdlib Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import JSON
+
+"""
+    gen( x, name )
+
+Generate fixture data and write to file.
+
+# Arguments
+
+* `x`: domain
+* `name::AbstractString`: output filename
+
+# Examples
+
+``` julia
+julia> x = range( -708, stop=709, length=2001 );
+julia> gen( x, \"data.json\" );
+```
+"""
+function gen( x, name )
+	y = cosc.( x );
```

Comment:
Why the period after `cosc`?

Reply:
I haven't used Julia before, but the version I have (1.0.3) needed that period there to enable the function to process vector/array inputs.

Reply:
Ah! Thanks! We have yet to upgrade to `v1` and I, at least, was unaware of this change. Good to know!